app/models/dimensions/edition.rb

```diff
@@ -23,6 +23,15 @@ class Dimensions::Edition < ApplicationRecord
   end
   scope :relevant_content, -> { where.not(document_type: %w[redirect gone]) }
 
+  def self.search(query)
+    sql = <<~SQL
+      to_tsvector('english',title) @@ plainto_tsquery('english', :q) or
+      to_tsvector('english',replace(base_path, '/', ' ')) @@ plainto_tsquery('english', :q2)
```

Comment:
Will using the 'English' dictionary' effect search with content written in other locales?

Reply:
For the time being we are focused on english. For other locales we won't have stemming, but it doesn't feel like something prioritised by the business. Something to discuss in the future, though.